yb-voyager/cmd/metadataDB.go

```diff
@@ -333,7 +333,7 @@ func (m *MetaDB) GetExportedEventsStatsForTable(schemaName string, tableName str
 	var inserts int64
 	var updates int64
 	var deletes int64
-	query := fmt.Sprintf(`select num_total, num_inserts, num_updates, num_deletes from %s WHERE schema_name='%s' AND table_name='%s'`,
+	query := fmt.Sprintf(`select SUM(num_total), SUM(num_inserts), SUM(num_updates), SUM(num_deletes) from %s WHERE LOWER(schema_name)=LOWER('%s') AND LOWER(table_name)=LOWER('%s')`,
```

Comment:
@sanyamsinghal for now, assumed that this will work FOR NOW, because we're not supporting case sensitivity anyway. So let me know!